charts/crljanitor/templates/secrets.yaml

```diff
@@ -59,11 +59,11 @@ spec:
 apiVersion: secrets-manager.tuenti.io/v1alpha1
 kind: SecretDefinition
 metadata:
-  name: secretdefinition-crl-janitor-app-sa
+  name: secretdefinition-crl-janitor-cloudsql-sa
   labels:
     {{ template "crljanitor.labels" . }}
 spec:
-  name: crl-janitor-sa
+  name: crl-janitor-cloudsql-sa
   keysMap:
     service-account.json:
       path: {{ .Values.vault.pathPrefix }}/crl_janitor/app-sa
```

Comment:
Is there a TODO here to create another service account in terraform and key in vault for this?

Reply:
Added a TODO. @gmalkov Hi Greg, I remembered you mentioned you are refactoring the SA to cloudsql-SA at some point? When that would be happen? Thanks